distributed_index_tests/conf/hosts.ini

```diff
@@ -1,8 +1,8 @@
 [vms]
 vm1=172.23.105.164
 vm2=172.23.105.165
-vm3=172.23.122.251
-vm4=172.23.122.250
+vm3=172.23.122.250
+vm4=172.23.122.251
```

Comment:
These are currently targeted by the Jenkins job. We need to update the jenkins job to generate an ini file from test before we ignore these